### DISCUSSION TITLE
Requirements not met

### DISCUSSION THREAD

**Discussion by romeero12** [Atlas-OS/Atlas#1276] [Help & Support]
Hello, how can i fix this?

Thanks.

![Captura de pantalla 2024-10-24 154719](https://github.com/user-attachments/assets/1586f89c-3077-4c97-aeda-53495f9b33b2)

**Comment by kamild1996** [comment]
You're not running the right version of Windows. Have a look at the documentation, mainly:
- https://docs.atlasos.net/getting-started/installation/ (had you followed this, you would certainly not have seen this error)
- https://docs.atlasos.net/install-faq/windows-version-support/

**Comment by itsnanobug** [comment]
I'm having this issue as well.
I tried to install it in a VM at first, before using a bare metal system.
The only reason I can think of, why it shouldn't work, is that it's a VM.

**Reply by kamild1996** [reply]
It has nothing to do with running in a VM. The only reason this error ever shows up is because the system's build number is not present on the list of builds supported by the playbook.

**Reply by itsnanobug** [reply]
I just realized that I shouldn't have updated Windows fully.
Is there any way I can make it run regardless of the build? It's a VM, so if it breaks, it's not going to matter much.

**Reply by LumiFae** [reply]
There is, but you'll have to figure it out yourself or join the discord and talk in the development chats and people will give advice there. If we tell you how to in this thread everyone will start doing it and be annoying us with questions.

**Reply by itsnanobug** [reply]
I get it. Thanks :)
